go.mod

```diff
@@ -223,4 +310,5 @@ require (
 replace (
 	github.com/agl/ed25519 => github.com/bnb-chain/edwards25519 v0.0.0-20200305024217-f36fc4b53d43
 	github.com/ethereum/go-ethereum => github.com/GOATNetwork/goat-geth v0.0.0-20240904132236-852e3254aff5
+	github.com/goatnetwork/goat => ../goat
```

Comment:
If this version is no longer available, you can go to github.com/goatnetwork/goat@main to get the main version locally and copy from the console. Pls don't use "replace ../"

Reply:
Thank you for your suggestion, I hava fix it